test/demo/expiry.inttest.ts

```diff
@@ -26,6 +28,7 @@ const options: NotificationOptions = {
 
 beforeEach(async () => {
     jest.resetAllMocks();
+    clearDatabase();
```

Comment:
This will only clear the database and so the store will theoretically will keep hold of any notifications unless you also use `clearStoredNotifications`.  Although, `clearStoredNotifications` would also clear the notifications table, but not the others.

Reply:
Hmmm. I think don't have this here at all then. Instead, makes sure that in the new test, the notifications are created with the same data they'd have if they'd been created by the test app set up in `beforeEach`. That way, anything leftover will be caught by the existing `clearStoredNotifications` call

Reply:
I have removed the `clearDatabase()`